website/server/models/user/hooks.js

```diff
@@ -59,6 +66,10 @@ function _populateDefaultTasks (user, taskTypes) {
         });
       }
 
+      if (taskDefaults.tags) {
+        newTask.tags = _.map(taskDefaults.tags, findTag);
```

Comment:
It appears that `_.map` is not affected significantly in the upgrade to v4, so everything should be good.